src/core/public/http/http_service.ts

```diff
@@ -45,6 +45,7 @@ export class HttpService implements CoreService<HttpSetup, HttpStart> {
     );
     const fetchService = new Fetch({ basePath, kibanaVersion });
     const loadingCount = this.loadingCount.setup({ fatalErrors });
+    loadingCount.addLoadingCountSource(fetchService.getRequestCount$());
```

Comment:
NIT: could add a test to ensure this is properly called.

Reply:
I added a test, but to avoid some really complex mocking I just asserted that `addLoadingCountSource` is called with an Observable instance, not specifically the one from `fetchService.getRequestCount$`. WDYT?

Reply:
Sorry, was under water last week on the SO management, did not see the notif. SGTM.